.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+
+version: 2
+updates:
+
+  - package-ecosystem: "cargo"
```

Comment:
I've been considering doing this for some other crates. I'll be watching to see how well this works for you!